### ISSUE TITLE
Wrong timezone for the %date% token

### ISSUE THREAD

**Issue by Leeiio** [Bionus/imgbrd-grabber#2220]
**Bug description**

My filename format is %date:format=yyyy-MM-dd hh.mm% - %md5%.%ext%, and I have downloaded an image named '2020-12-29 17.08 - 55e79668de6ea92086cdc27671adca5b.jpg', but the image file's created date is 2020-12-30 01.08, it's a tomorrow date :) I am in +8 timezone.

**Steps to reproduce**

none.

**Expected behavior**

Please fix the timezone issue of created date.

**Context**

none.

**Screenshots**

none.

**System information**

- OS: Windows 10
- Grabber version: 7.3.2

**Comment by Bionus** [comment]
So if I understand correctly, the %date%  in the filename has the correct value, but the one in the filename is UTC treated as the local timezone? (so an 8h delay?)

Which source did it happen on?
I just tried on Danbooru, and even with a timezone change (-5 to +1) I got the correct creation date 🤔 

However, the `%date%` used the original time with the original timezone (so -5).
I guess it should be changed to use UTC or the local timezone by default instead, with a filename option to choose another timezone (or a real option I guess).
